landoapi/models/secapproval.py

```diff
@@ -0,0 +1,61 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+"""Models related to the sec-approval process.
+
+See See https://wiki.mozilla.org/Security/Bug_Approval_Process.
+"""
+
+from sqlalchemy.dialects.postgresql.json import JSONB
+
+from landoapi.phabricator import PhabricatorClient
+from landoapi.storage import db
+
+
+class SecApprovalRequest(db.Model):
+    """Represents an event that added a sec-approval comment to a revision."""
+
+    __tablename__ = "secapproval_requests"
+
+    id = db.Column(db.Integer, primary_key=True)
+
+    # The revision ID that this event applies to.
+    revision_id = db.Column(db.Integer, nullable=False)
```

Comment:
Probably worth sticking an index on this column since it'll be queried to find specific rows. a `HASH` index is probably the most efficient here, but the default index type would also be fine.